source/gateway_test.go

```diff
@@ -63,7 +63,7 @@ func (suite *GatewaySuite) SetupTest() {
 	suite.source, err = NewIstioGatewaySource(
```

Comment:
Every feature needs to have tests included, only change types is not good enough. Please add some different tests.